Description: Correct pod NAME issue in LedgerSMB/Template/HTML.pm
 Is missing the dash from the pod NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472621&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Template/HTML.pm
+++ b/LedgerSMB/Template/HTML.pm
@@ -1,7 +1,7 @@
 
 =head1 NAME
 
-LedgerSMB::Template::HTML  Template support module for LedgerSMB
+LedgerSMB::Template::HTML - Template support module for LedgerSMB
 
 =head1 METHODS
 
